llvm/lib/Analysis/ScalarEvolution.cpp

```diff
@@ -6019,6 +6019,37 @@ const SCEV *ScalarEvolution::createNodeFromSelectLikePHI(PHINode *PN) {
   return nullptr;
 }
 
+// Returns SCEV for the first operand of a phi if all phi operands have
```

Comment:
```suggestion
/// Returns SCEV for the first operand of a phi if all phi operands have
```
Use `///` for header comments.